src/vmm/src/lib.rs

```diff
@@ -386,25 +373,35 @@ impl VMM {
 mod tests {
     use super::*;
 
-    use std::io::ErrorKind;
     use std::path::PathBuf;
 
     use linux_loader::loader::bootparam::setup_header;
     use linux_loader::loader::elf::PvhBootCapability;
+    use linux_loader::loader::elf::PvhBootCapability::PvhEntryNotPresent;
```

Comment:
nit: merge this `use` with the one above

Reply:
Done.